argo-cd-apps/overlays/development/repo-overlay.yaml

```diff
@@ -74,6 +74,7 @@ spec:
   source:
     repoURL: https://github.com/redhat-appstudio/infra-deployments.git
     targetRevision: main
+    path: components/spi/overlays/development
```

Comment:
not needed here, since it's patch so only differences should be here - targeRevios and repoURL are here as placeholders which are replaced by scripts which are called by preview mode.